### PR TITLE
Lena/events post web

### PR DESCRIPTION
**Pull request by LenaNguyen**
Created the update events endpoint. When editing the details of an event and saving, the page should reload and display the updated event.

### CODE REVIEW

File: src/api/eventsAPI.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { get } from './ApiUtils';
```
Comment (LeozMaxwellJilliams) [inline src/api/eventsAPI.js]: Can we move this file to eventApi.js?

File: src/components/events/EventForm.jsx [hunk 2]
```diff
@@ -58,14 +60,15 @@ class EventForm extends React.Component {
...
+      eventDate: moment(eventDate).format('YYYY-MM-DD HH:mm:ss.SSS'),
       eventTitle,
       numJudges
     };
     if (formType === dialogType.NEW) {
       await addData(collectionName, data);
     } else {
-      await updateData(collectionName, eventId, data);
+	  await updateEvent(eventId, data);
+	  window.location.reload();
```
Comment (bmlunchbox) [inline src/components/events/EventForm.jsx]: - add callback function to your API call
- resolve with .then
- manage page reloads through setState -> this updates the data and forces a rerender with the new data

File: src/components/events/EventsSection.jsx [hunk 3]
```diff
@@ -18,31 +20,18 @@ class EventsSection extends React.Component {
...
-        const event = {
-          eventDate: date,
-          eventTitle,
-          id: doc.id,
-          numJudges
-        };
-        events.push(event);
-      });
+    getEvents().then((response) => {
+      const events = Object.values(response.data);
```
Comment (bmlunchbox) [inline src/components/events/EventsSection.jsx]: follow this format

File: src/components/events/EventForm.jsx [hunk 4]
```diff
@@ -6,24 +6,25 @@ import { withStyles } from '@material-ui/core/styles';
...
   constructor(props) {
     super(props);
     const { currentValues } = props;
 
     this.state = {
       disabledSave: true,
-      eventDate: currentValues.eventDate || moment().format('DD/MM/YYYY'),
-      eventTitle: currentValues.eventTitle || '',
-      numJudges: currentValues.numJudges || 0,
+      eventDate: currentValues.event_date || moment().format('DD/MM/YYYY'),
```
Comment (imruahmed) [inline src/components/events/EventForm.jsx]: Why did we switch to snake case?

File: src/components/events/EventTableRow.jsx [hunk 5]
```diff
@@ -11,14 +12,15 @@ class EventTableRow extends React.Component {
 
   render() {
     const {
-      currentValues,
-      id
+	  currentValues,
```
Comment (imruahmed) [inline src/components/events/EventTableRow.jsx]: wierd spacing

File: src/components/events/EventTableRow.jsx [hunk 6]
```diff
@@ -11,14 +12,15 @@ class EventTableRow extends React.Component {
...
+	  onUpdate,
+      id,
     } = this.props;
 
     const {
-      eventTitle,
-      eventDate,
-      numJudges
+      event_title,
```
Comment (imruahmed) [inline src/components/events/EventTableRow.jsx]: snake case??

File: src/components/events/EventsSection.jsx [hunk 7]
```diff
@@ -14,35 +15,30 @@ class EventsSection extends React.Component {
     this.state = {
       events: null,
       loading: true
-    };
+	};
```
Comment (imruahmed) [inline src/components/events/EventsSection.jsx]: spacing

**Pull request closed without merging**